src/main/java/com/emilpiekos/App.java

```diff
@@ -1,13 +1,36 @@
 package com.emilpiekos;
 
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+import java.security.InvalidKeyException;
+import java.sql.SQLException;
+
+public class App {
+    public static void main(String[] args) throws SQLException {
+        TransactionDao tdao = new TransactionDao();
+
+        while (true) {
+            UsersInOut.printOptions();
+            Integer option = UsersInOut.inputNumber();
+            switch (option) {
+                case 1 -> {
+                    tdao.insertTransaction();
+                }
+                case 2 -> {
+                    tdao.updateTransaction();
+                }
+                case 3 -> {
+                    tdao.deleteTransaction();
+                }
+                case 4 -> {
+                    tdao.showTransactions("income");
+                }
+                case 5 -> {
+                    tdao.showTransactions("outgo");
+                }
+                case 6 -> {
+                    return;
+                }
+                default -> throw new RuntimeException("Invalid option");
```

Comment:
Rzucenie wyjątku to nie jest w tym przypadku najlepsze rozwiązanie, bo aplikacja zakończy się błędem. Lepiej byłoby tutaj wyświetlić komunikat o błędnej opcji, albo po prostu zakończyć aplikację, jak w przypadku opcji 6